dataactbroker/handlers/fileHandler.py

```diff
@@ -644,7 +650,12 @@ def generateD2File(self):
         # Generate and upload d2 file to S3
         user_id = LoginSession.getName(session)
         timestamped_name = s3UrlHandler.getTimestampedFilename(CONFIG_BROKER["award_file_name"])
-        upload_file_name = "".join([str(user_id), "/", timestamped_name])
+
+        if self.isLocal:
+            upload_file_name = "".join([CONFIG_BROKER['broker_files'], timestamped_name])
```

Comment:
Generally local files are stored there (based on how submit files works locally) so that is why this follows the same model. But that can be altered.